src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -199,6 +201,12 @@ const ProjectSelector = createReactClass({
   getProjectNode(team, project, highlightText, hasSingleTeam, isSelected) {
     let projectId = project.slug;
     let label = this.getProjectLabel(team, project, hasSingleTeam, highlightText);
+    let platforms;
+    if (typeof project.platforms !== 'undefined') {
+      platforms = project && project.platforms.slice(0, 3);
```

Comment:
`slice` is unnecessary since `PlatformList` should determine how many to display.